authzed/api/v1/core.proto

```diff
@@ -64,6 +64,37 @@ message ObjectReference {
   } ];
 }
 
+// Consistency will define how a request is handled by the backend.
+// By defining a consistency requirement, and a token at which those
+// requirements should be applied, where applicable.
+message Consistency {
```

Comment:
I actually think that moving this might break Java

Reply:
really? how so?

Reply:
I see that `Core` shows up in the package hierarchy for java. Removed. We'll move it in v2.